AdminServer/appscale/admin/push_worker_manager.py

```diff
@@ -94,6 +95,19 @@ def update_worker(self, queue_config):
     yield gen.with_timeout(timedelta(seconds=60), start_future,
                            IOLoop.current())
 
+  @gen.coroutine
+  def stop_worker(self):
+    """ Removes the monit configuration for the project's push worker. """
+    status = yield self._wait_for_stable_state()
+    if status == MonitStates.RUNNING:
+      logging.info('Stopping push worker for {}.'.format(self.project_id))
+      yield self.monit_operator.send_command(self.monit_watch, 'stop')
+      watch_file = '{}/appscale-{}.cfg'.format(MONIT_CONFIG_DIR, self.monit_watch)
+      os.remove(watch_file)
+    else:
+      logging.debug('Not stopping push worker for {} since it is not running.'.format(self.project_id))
```

Comment:
Since https://github.com/AppScale/appscale/pull/2546 has been merged, use `logger` instead of `logging`. You'll need to do a merge commit in order for it to work since this branch is based off an older one.